snuba/web/views.py

```diff
@@ -92,6 +92,24 @@ def check_clickhouse() -> bool:
         return False
 
 
+def drop_dataset(dataset: Dataset) -> None:
+    for storage in dataset.get_all_storages():
+        cluster = storage.get_cluster()
+        clickhouse = cluster.get_query_connection(ClickhouseClientSettings.MIGRATE)
+        database = cluster.get_database()
+
+        tables_to_empty = {
+            schema.get_local_table_name()
+            for schema in storage.get_schemas().get_unique_schemas()
+            if isinstance(schema, TableSchema)
+        }
+
+        for table in tables_to_empty:
+            clickhouse.execute(f"TRUNCATE TABLE IF EXISTS {database}.{table}")
+
+    redis_client.flushdb()
```

Comment:
I would keep this in the method drop where it was before. Redis contains data that is not dataset specific. When I invoke a "trunncate_dataset" method I would expect only the database tables to be wiped.